src/niscope/metadata/functions_addon.py

```diff
@@ -487,12 +488,38 @@
             'note': 'Some functionality, such as time stamping, is not supported in all digitizers.',
         },
     },
+    'FancyGetEqualizationFilterCoefficients': {
+        'codegen_method': 'python-only',
+        'returns': 'ViStatus',
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'vi',
+                'type': 'ViSession',
+                'documentation': {
+                    'description': 'The instrument handle you obtain from niScope_init that identifies a particular instrument session.',
+                },
+            },
+            {
+                'direction': 'in',
+                'name': 'channel',
+                'type': 'ViChar[]',
+                'documentation': {
+                    'description': 'The channel to configure.',
+                },
+            },
+        ],
+        'documentation': {
+            'description': 'Retrieves the custom coefficients for the equalization FIR filter on the device. This filter is designed to compensate the input signal for artifacts introduced to the signal outside of the digitizer. Because this filter is a generic FIR filter, any coefficients are valid. Coefficient values should be between +1 and –1.',
```

Comment:
Isn't this string identical to the one in the extracted metadata? It'd be great if we could just reference that rather than copy it.

Reply:
The problem is we don't have access to the the function dictionary in functions_addon. It is in \_\_init\_\_ that we load functions.py and then load functions_addon.py and then call merge.

I could add a line in \_\_init\_\_.py that looks like 
``` python
functions['funcA']['documentation']['description'] = functions['funcB']['documentation']['description']
```
but that means it will be separated from all the other addon metadata. It seems less bad to duplicate the description in the addon file.